pkg/fmmon/analyze.go

```diff
@@ -0,0 +1,46 @@
+package fmmon
+
+import (
+	"log"
+	"strconv"
+
+	"github.com/goodbuns/fishmon/pkg/adafruitio"
+)
+
+// Analyze analyzes feed information and alerts on any issues.
+func Analyze(feeds *[]adafruitio.Feed, alerts chan string, numFeeds int, minTemp, maxTemp float64, webhookURL string, update bool) {
+	// Check number of feeds.
+	roleCall := false
+	if len(*feeds) < numFeeds {
+		m := "expected " + strconv.Itoa(numFeeds) + " feeds, got only " + strconv.Itoa(len(*feeds))
+		alerts <- m
+		roleCall = true
+	}
+
+	// Check temperatures of feeds.
+	u := ""
+	for _, feed := range *feeds {
+		if roleCall {
+			m := feed.Name + " still online"
+			alerts <- m
+		}
+		temp, err := strconv.ParseFloat(feed.LastValue, 64)
```

Comment:
You can define `LastValue float64` in the `Feed struct`, and `encoding/json.Unmarshal` will do this for you.